Project.toml

```diff
@@ -24,7 +24,6 @@ FileIO = "1"
 GLMakie = "~0.10"
 GeometryBasics = "0.4.10"
 Joysticks = "0.1.2, 0.1.4"
-KiteUtils = "0.6, 0.7"
```

Comment:
You cannot delete this, I cannot make a new release without an upper limit of the version number.